Library/Layers/Foundation Layer/HealthClientHandler.swift

```diff
@@ -0,0 +1,77 @@
+/*
+* Copyright (c) 2024, Nordic Semiconductor
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without modification,
+* are permitted provided that the following conditions are met:
+*
+* 1. Redistributions of source code must retain the above copyright notice, this
+*    list of conditions and the following disclaimer.
+*
+* 2. Redistributions in binary form must reproduce the above copyright notice, this
+*    list of conditions and the following disclaimer in the documentation and/or
+*    other materials provided with the distribution.
+*
+* 3. Neither the name of the copyright holder nor the names of its contributors may
+*    be used to endorse or promote products derived from this software without
+*    specific prior written permission.
+*
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+* ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+* WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+* IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+* INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+* NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+* PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+* WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+* ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*
+* Created by Jules DOMMARTIN on 04/11/2024.
+*/
+
+class HealthClientHandler: ModelDelegate {
+    var messageTypes: [UInt32 : MeshMessage.Type]
+    
+    var isSubscriptionSupported: Bool = false
+    
+    var publicationMessageComposer: MessageComposer? {
```

Comment:
This can be set like here:
https://github.com/NordicSemiconductor/IOS-nRF-Mesh-Library/blob/28ff22e8ad636c664e91ceceaef8796ce1a33633/Library/Layers/Foundation%20Layer/PrivateBeaconClientHandler.swift#L45

Reply:
https://github.com/NordicSemiconductor/IOS-nRF-Mesh-Library/blob/fd8e5252e26bd62b2c746c3e5a35a1215edcc039/Library/Layers/Foundation%20Layer/HealthClientHandler.swift#L38